src/test/kotlin/net/maizegenetics/phgv2/cli/Hvcf2GvcfTest.kt

```diff
@@ -0,0 +1,160 @@
+        assertEquals(0, result.statusCode )
+
+        // Verify the tiledbUri/reference folder exists and contains the ranges file
+        val referenceDir = "${dbPath}/reference/"
+
+        // RUn alignAssemblies test to get MAF files.
+        // Run createMAFVCf on the assemblies LineA and LIneB to get
+        // data into the db.
+
+        println("testSimpleHvcf2Gvcf: running AlignAssemblies")
+        val alignAssemblies = AlignAssemblies()
+
+        result = alignAssemblies.test(
+            "--gff ${TestExtension.smallseqAnchorsGffFile} --reference-file ${TestExtension.smallseqRefFile} " +
+                    "--assembly-file-list ${TestExtension.smallseqAssembliesListFile} -o ${TestExtension.tempDir} --total-threads 1 --in-parallel 1"
+        )
+
+        println("testSimpleHvcf2Gvcf: result output: ${result.output}")
+        assertEquals(result.statusCode, 0, "status code not 0: ${result.statusCode}")
+
+        // Load assemblies using CreateMafVcf - creates and loads gvcf and hvcf
+        // remember - there is no gvcf for the ref
+        println("testSimpleHvcf2Gvcf: running CreateMafVcf")
+        val createMafVcf = CreateMafVcf()
+        result = createMafVcf.test("--db-path ${dbPath} --bed ${ranges} --reference-file ${refFasta} --maf-dir ${TestExtension.tempDir} -o ${TestExtension.testVCFDir}")
+        println(result.output)
+
+        // Need to load the vcf now!
+        // Load the vcf files into the tiledb dataset
+        println("testSimpleHvcf2Gvcf: running LoadVcf")
+        val loadVcf = LoadVcf()
+        result = loadVcf.test("--db-path ${dbPath} --vcf-dir ${TestExtension.testVCFDir}")
+
+        // verify the gvcf and hvcf files were created, and that the hvcf file was written somewhere -
+        // are we doing that by default?  if so, where?  Is it undewr the reference folder?
+        // RUn to here, stop, verify the output files.  It is written to the tempDir/vcfDir (${TestExtension.testVCFDir})
+        // so copy LineB.h.vcf.gz to LineBPath.h.vcf.gz
+        // then run hvcf2gvcf on LineBPath.h.vcf.gz
+
+        // Copy the $TestExtension.testVCFDir/LineB.h.vcf.gz to $TestExtension.testVCFDir/LineBPath.h.vcf.gz
+        // Make directory ${TestExtension.testVCFDir}/testOutputGVCFDir
+
+        println("NOW ... running hvcf2gvcf")
+        val testGVCFdir = "${TestExtension.testVCFDir}/testOutputGVCFDir"
+        File(testGVCFdir).mkdirs()
+        val lineBPathHvcf = "${testGVCFdir}/LineBPath.h.vcf.gz"
+        val lineBHvcf = "${TestExtension.testVCFDir}/LineB.h.vcf.gz"
+        File(lineBHvcf).copyTo(File(lineBPathHvcf))
+
+        // Run hvcf2gvcf on the copied file
+        val hvcf2gvcf = Hvcf2Gvcf()
+        result = hvcf2gvcf.test("--db-path ${dbPath} --hvcf-dir $testGVCFdir --output-dir ${testGVCFdir} --reference-file ${refFasta}")
+
+        // TODO
+        // Need some assertions here
+        println("done !!")
+
```

Comment:
missing assertions

Reply:
yup - it's in the TODO.   - working on it.